client/i18n/i18next.ts

```diff
@@ -20,6 +21,16 @@ const isDev = __WEBPACK_ENV.NODE_ENV !== 'production'
  */
 export type TransInterpolation = any
 
+/**
+ * The locale that was reported to us by the user's browser. This locale can be overwritten by
+ * user's explicit choice in the top-links dropdown. We send this locale to the server during
+ * login/signup/getCurrentSession actions.
+ *
+ * NOTE(2Pac): Session storage gets cleared after logging in, so be careful about using this only in
```

Comment:
We should probably stop doing that tbh, especially if we're storing anything else in there. I don't think the `storageKey` stuff really needs to be on `SessionStorage` (sharing it between tabs is irrelevant), it could just be a global `Map`